fdbserver/DataDistribution.actor.cpp

```diff
@@ -2470,6 +2486,11 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 			server_info[newServer.id()] = r;
 			// Establish the relation between server and machine
 			checkAndCreateMachine(r);
+			// Add storage server to pid map
+			ASSERT(r->lastKnownInterface.locality.processId().present());
+			StringRef pid = r->lastKnownInterface.locality.processId().get();
+			pid2server_info[pid].push_back(r);
+			canStartStorageWiggling.trigger();
```

Comment:
This may be inconsequential, but it seems like it could be a performance improvement to only wake the perpetualStorageWiggler if the pid is the same as the wiggler's current pid? Like how the TSS check compares the pid to the wiggle's current pid. Or is there some race I'm missing where that could cause the wiggle to miss a process?

Reply:
Yes. I think this can be improved, even not use this trigger.